typings/balena-sdk.d.ts

```diff
@@ -207,7 +207,7 @@ declare namespace BalenaSdk {
 			ApplicationMember
 		>;
 		// this is what the api route returns
-		social_service_account: ReverseNavigationResource<SocialServiceAccount>;
+		social_service_account: ReverseNavigationResource<Pick<SocialServiceAccount, 'provider' | 'display_name'>>;
```

Comment:
I think we could just drop all other props from SocialServiceAccount, instead of using a Pick here.

Reply:
It's your call @thgreasi

Reply:
@Page- I just confirmed this locally on the UI. I indeed prefer just having an accurate SocialServiceAccount instead of a Pick. Let me know if you prefer me to pick this up.

Reply:
done